docs/devguide/run_time_configuration.rst

```diff
@@ -1103,22 +1106,22 @@ Those properties, along with options specific to OpenDDS's RTPS discovery implem
   .. prop:: ResendPeriod=<sec>
     :default: ``30``
 
-    The number of seconds that a process waits between the announcement of participants (see :omgspec:`rtps:8.5.3`).
+    The number of seconds that a process waits between the :ref:`SPDP participants announcements <spdp>`.
```

Comment:
```suggestion
    The number of seconds that a process waits between the :ref:`SPDP participant announcements <spdp>`.
```

Reply:
Thanks, my spellcheck doesn't check the contents of links... I should probably find a workaround for that.